modules/installation-aws-add-local-zone-locations.adoc

```diff
@@ -39,3 +57,7 @@ $ aws ec2 modify-availability-zone-group \
     --group-name "<value_of_GroupName>" \
     --opt-in-status opted-in
 ----
++
+where:
++
+<value_of_GroupName>:: Specifies the name of the group of the Local Zone you want to create subnets on. For example, specify `us-east-1-nyc-1` to use the zone `us-east-1-nyc-1a`, US East (New York).
```

Comment:
Instead of making this a list-type item, can it be a callout in the code? So put a <1> on line 57 then list this info as a callout?